### PR TITLE
Create Lokole admin user during setup

### PR DESCRIPTION
**Pull request by c-w**
### Description of changes proposed in this pull request.

Add a default admin account when setting up Lokole as per https://github.com/iiab/iiab/pull/1305#discussion_r236069663.

### Smoke-tested in operating system.

By @aidan-fitz 

### Mention a team member for further information or comment using @ name

@holta

### CODE REVIEW

**Comment by holta** [comment]
Is this (PR #1317) tested / safe to merge?

PS i can also merge PR #1305 if you think that documentation's solid enough?

**Comment by holta** [comment]
> Is this (PR #1317) tested / safe to merge?
> 
> PS i can also merge PR #1305 if you think that documentation's solid enough?

@c-w & @aidan-fitz please express an opinion before today if possible, prior to tmrw morning's IIAB call when the final outlines of IIAB 6.7 will start taking shape we hope!

**Comment by c-w** [comment]
@aidan-fitz

**Comment by holta** [comment]
@aidan-fitz take some time off for exam week and get back in touch when ya can mid/late December :-)

**Comment by veeara282** [comment]
Hi @c-w and @holta, sorry I didn't see this earlier. You're right, we've been super busy with finals.... Angela and I will take a look after our finals.

File: roles/lokole/defaults/main.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Info needed to install Lokole
-lokole_version: "0.1.24"
+lokole_version: "0.1.26"
+lokole_admin_user: admin
```
Comment (holta) [inline roles/lokole/defaults/main.yml:3]: @c-w @aidan-fitz

IIAB's universal recommendation for OOB login to apps is: Admin/changeme

(Can we specify the username start with an uppercase `A` above, to be sure?)

**Comment by holta** [comment]
Merging tentatively to encourage further testing -- please help when you can, thanks @aidan-fitz & All!

Ref #1293

File: roles/lokole/defaults/main.yml (hunk 1 above)
Comment (veeara282) [reply roles/lokole/defaults/main.yml:3]: I made this change to branch `add-lokole-admin-user`.

**Pull request merged**